database/seedReviews.js

```diff
@@ -0,0 +1,32 @@
+const mysql = require('mysql');
+const faker = require('faker');
+
+const connection = mysql.createConnection({
```

Comment:
Can you import the connection made in `database/index.js`? Same goes for `seedUsers.js`.